### DISCUSSION TITLE
Expenses - Sales Tax Paid - US

### DISCUSSION THREAD

**Discussion by hiltonmw** [firefly-iii/firefly-iii#7783] [General]
I'm new to this and I'm not an accountant.  I am looking to use FF3 to help with expenses.

I need to figure out how much sales tax was paid out on purchases / expenses and I'm not sure how to go about doing it.

I can enter the $ amount for the expense, but I don't seem to see any special fields or ways to track the sales tax / taxes that were paid on each expense.

For example, I need to report that $10.60 was expensed, where $10.00 was the sale price, and $0.60 was tax.

Am I missing something?  Would I use a "split" on the transaction and create a separate "line item" with separate tags/categories?

Thanks!

**Comment by JC5** [comment]
There are no fields to track the (sales) tax, no. You could add another split with that information though.

**Comment by hiltonmw** [comment]
Got it!  That's what I'm doing now.  Thanks.
